applications/GeoMechanicsApplication/tests/lysmer_boundary_column2d_quad/README.md

```diff
@@ -0,0 +1,34 @@
+# Lysmer absorbing boundary conditions on a column made of rectangulars
+
+This test verifies the implementation of the Lysmer absorbing boundary conditions. 
+
+## Setup
+
+The test is performed for an 1D column that is made with 2D rectangular elements. The column height and width are 10,000 mm and 1,000 m, respectively.
+The column consists of ten 4-noded elements (using the UPwSmallStrainElement2D4N class) of 1,000mm x 1,000 mm. A
+schematic representation can be found in the figure below:
+
+![MeshStructure](MeshStructure.svg)
+
+All Nodes have fixed displacement in the X direction. 
+
+An instant load of -10 N is applied in the Y direction at the column top that is defined by Nodes 1 and 2. 
+
+The absorbing boundary conditions are applied on a line created by Nodes 21 and 22. The unity absorbing factors are used there.
+
+The material is described using:
+
+-   A linear elastic plane stress model (LinearElasticPlaneStrain2DLaw),
+-   Young's modulus is 10,000 N/mm2
+-   Poisson's ratio is 0.2,
+-   A thickness of 1 mm,
+-   Density of solid is 2.65 kg/mm3,
+-   Porosity of 0.3.
+
+
+## Assertions
+
+The instant load generates a P-wave. The test 
+
+-   calculates time when the P-wave arrives at certain coordinates (Nodes 5, 11, and 17) inside the column and
+-   checks the velocity of the corresponding nodes afterwards. 
```

Comment:
I think it might be nice to add the formulas here that defined in the `calculate_expected_p_wave_velocity` and `calculate_expected_velocity_column` function, so we can see without looking at the python file what the expected velocities are.

Reply:
Equations are added now.